jaxrs/src/main/java/com/cerner/beadledom/jaxrs/provider/CorrelationIdFilter.java

```diff
@@ -33,6 +33,7 @@
   private static final String DEFAULT_MDC_NAME = "Correlation-Id";
```

Comment:
Should we change the default header name in 4.0 to match our internal usage?

Reply:
We already do that in our internal module, CernerResteasyModule. I don't think the open-source beadledom would want the name to include `Cerner`